accumulator_updater/programs/accumulator_updater/src/state/whitelist.rs

```diff
@@ -47,7 +47,7 @@ pub struct WhitelistVerifier<'info> {
         seeds = [b"accumulator".as_ref(), b"whitelist".as_ref()],
         bump = whitelist.bump,
     )]
-    pub whitelist:  Account<'info, Whitelist>,
+    pub whitelist:  Box<Account<'info, Whitelist>>,
```

Comment:
why did this change?

Reply:
using a `Box` will move the account to the heap which preserves more space for the stack . Heap gets allocated 32KB by default - https://github.com/solana-developers/anchor-zero-copy-example/blob/main/README.md#explanation-of-solana-memory-and-zero-copy

Reply:
added comment to clarify reasoning for this